ethportal-api/src/types/content_key.rs

```diff
@@ -27,6 +27,8 @@ pub enum HistoryContentKey {
     BlockReceipts(BlockReceiptsKey),
     /// An epoch header accumulator.
     EpochAccumulator(EpochAccumulatorKey),
+    /// Unknown content key value
+    Unknown(H256),
```

Comment:
isn't `H256` constrained to 32 bytes?  Should this be a `Vec<u8>` to accommodate longer values?

Reply:
Good catch, updated!